src/websocket/client.ts

```diff
@@ -12,10 +12,12 @@ import { AppSignal, AppSignalCb, SignalResponseGeneric } from '../api/app'
 export class WsClient {
   socket: Websocket
   pendingRequests: Record<string, { fulfill: Function }>
+  noSignalCbFlag: Boolean
```

Comment:
Suggesting a longer variable name and changing the type from `Boolean` (a class) to `boolean` (typescript primitive)
```suggestion
  alreadyWarnedNoSignalCb: boolean
```